Python/opcode_metadata.h

```diff
@@ -1,5 +1,6 @@
 // This file is generated by Tools/cases_generator/generate_cases.py --metadata
```

Comment:
```suggestion
// This file is generated by Tools/cases_generator/generate_cases.py
```
While you're here, let's correct this comment (the `--metadata` flag works differently now).